.github/pull-request-template.md

```diff
@@ -8,11 +8,11 @@
 
 ### Initial checklist
 
-*   [ ] I read the support docs <!-- https://github.com/rehypejs/.github/blob/main/support.md -->
-*   [ ] I read the contributing guide <!-- https://github.com/rehypejs/.github/blob/main/contributing.md -->
-*   [ ] I agree to follow the code of conduct <!-- https://github.com/rehypejs/.github/blob/main/code-of-conduct.md -->
-*   [ ] I searched issues and couldn’t find anything (or linked relevant results below) <!-- https://github.com/search?q=user%3Arehypejs&type=Issues -->
-*   [ ] If applicable, I’ve added docs and tests
+- [ ] I read the support docs <!-- https://github.com/rehypejs/.github/blob/main/support.md -->
+- [ ] I read the contributing guide <!-- https://github.com/rehypejs/.github/blob/main/contributing.md -->
+- [ ] I agree to follow the code of conduct <!-- https://github.com/rehypejs/.github/blob/main/code-of-conduct.md -->
+- [ ] I searched issues and couldn’t find anything (or linked relevant results below) <!-- https://github.com/search?q=user%3Arehypejs&type=Issues -->
+- [ ] If applicable, I’ve added docs and tests
```

Comment:
`npm run format` should revert this change?

Reply:
prettier changed this before I added a .prettierignore, remark appears to be ignoring it

Reply:
manually fixed